plat/hisilicon/hikey/include/hikey_layout.h

```diff
@@ -58,10 +58,10 @@
  * + loader +
  * ++++++++++ 0xF980_1000
  * +  BL2   +
- * ++++++++++ 0xF981_8000
+ * ++++++++++ 0xF985_8000
  */
 #define BL2_BASE			(BL1_RO_BASE)		/* 0xf980_1000 */
-#define BL2_LIMIT			(0xF9818000)		/* 0xf981_8000 */
+#define BL2_LIMIT			(0xF9858000)		/* 0xf985_8000 */
```

Comment:
Original limit is not big enough for TBB?

Reply:
Correct!